code/WorkInProgress/construction/tools.dm

```diff
@@ -452,7 +452,7 @@
 
 	var/selecting = 0
 	var/mode = "floors"
-	var/icons = list("floors" = 'icons/turf/construction_floors.dmi', "walls" = 'icons/turf/construction_walls.dmi')
+	var/icons = list("floors" = 'icons/turf/conworker_floors.dmi', "walls" = 'icons/turf/conworker_walls.dmi')
```

Comment:
don't like the naming of this, being tied to the job

Reply:
hope the new ones are better